### PR TITLE
Adding voiceovers

### PR DESCRIPTION
**Pull request by audstav**
@DubeySandeep PTAL

Video is not yet done, will be completed in the next few days. 

Updated changes on:
https://oppia-user-guide.readthedocs.io/en/latest/index.html

### CODE REVIEW

**Comment by audstav** [comment]
@DubeySandeep I've updated this PR now with a video and also edited the index page, PTAL!

File: docs/contributor/voiceover.rst [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+.. |submit| image:: /images/submit_button.png
+            :alt: Submit button
+            :scale: 45 %
+
+.. note::
+
+   Currently Oppia only supports the MP3 file format for the audio file.
+
+6. Your application has now been sent for review. You will not be able to submit any more voiceovers for this or any other lesson until your review is accepted.
```
Comment (DubeySandeep) [inline docs/contributor/voiceover.rst]: application is accepted, right?

File: docs/contributor/voiceover.rst [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+   :alt: Voiceover accepted status
+
+   *Fig. 3*
+
+.. _recordvoiceover:
+
+Recording voiceovers
+----------------------
+
+1. To begin recording a voiceover, click the **Voiceover** button next to your accepted application card in the **My Contribution** page. You will be taken to the **Translation** screen.
```
Comment (DubeySandeep) [inline docs/contributor/voiceover.rst]: This is going to be changed as well as, users will have to claim a task before doing this. There is going to be a "Claim a task" step between "Applying to do a voiceover" & "Recording voiceovers".

File: docs/contributor/voiceover.rst [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+Watch the video to learn how to contribute voiceovers, or if you prefer reading the instructions instead, keep scrolling down.
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/0LabiUz9hp4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+.. _voiceoverapplication:
+
+Applying to do a voiceover
```
Comment (DubeySandeep) [inline docs/contributor/voiceover.rst]: Applying to become a voice-artist* (here and elsewhere)

**Review by DubeySandeep — COMMENTED** [review]
@audstav, Sorry for the late review I had missed this PR, I've left a few comments PTAL!

File: docs/contributor/voiceover.rst (hunk 1 above)
Comment (audstav) [reply docs/contributor/voiceover.rst]: Done

File: docs/contributor/voiceover.rst (hunk 3 above)
Comment (audstav) [reply docs/contributor/voiceover.rst]: Done

File: docs/contributor/voiceover.rst (hunk 2 above)
Comment (audstav) [reply docs/contributor/voiceover.rst]: I added this in, will change screenshots once this is available. I also assumed that the '**Translation**' screen in the lesson editor page will be '**Voiceover**', so I made that change as well. Let me know if not correct...

**Comment by audstav** [comment]
@DubeySandeep PTAL

**Review by DubeySandeep — APPROVED** [review]
LGTM!

**Pull request merged**